code/game/mecha/mecha_defense.dm

```diff
@@ -253,12 +253,12 @@
 				to_chat(user, "<span class='warning'>The welder must be on for this task!</span>")
 				return 1
 		else
-			to_chat(user, "<span class='warning'>The [name] is at full integrity!</span>")
+			to_chat(user, "<span class='warning'>[src] is at full integrity!</span>")
 		return 1
 
 	else if(istype(W, /obj/item/mecha_parts/mecha_tracking))
 		if(!user.transferItemToLoc(W, src))
-			to_chat(user, "<span class='warning'>\the [W] is stuck to your hand, you cannot put it in \the [src]!</span>")
+			to_chat(user, "<span class='warning'>[W] is stuck to your hand, you cannot put it in [src]!</span>")
```

Comment:
I'm pretty sure mecha are capitalized and thus proper by default. you may need better handling here, since they can be renamed and there is no way to account for that without better handling.
`you cannot put it in the Durand`
`you cannot put it in Durand`
`you cannot put it in the NERDSMASHER`
`you cannot put it in NERDSMASHER`